src/ImageSharp/ImageFrameCollection.cs

```diff
@@ -122,7 +122,7 @@ public abstract class ImageFrameCollection : IEnumerable<ImageFrame>
         public IEnumerator<ImageFrame> GetEnumerator() => this.NonGenericGetEnumerator();
 
         /// <inheritdoc/>
-        IEnumerator IEnumerable.GetEnumerator() => this.NonGenericGetEnumerator();
+        IEnumerator IEnumerable.GetEnumerator() => this.GetEnumerator();
```

Comment:
Best to call the method above to ensure no diversion is possible in future refactoring.